server/services/gameCreate.js

```diff
@@ -59,12 +59,18 @@ module.exports = class GameCreateService {
             settings
         });
 
-        // Calculate how many stars we need.
+        let isCustomGalaxy = game.settings.galaxy.galaxyType === 'custom';
```

Comment:
Would be better as a function in `gameService`. i.e `this.gameService.isCustomGalaxy(game)`

Reply:
looks like you already changed this?